tests/utilities/test_cli.py

```diff
@@ -1547,3 +1549,54 @@ def test_cli_logger_shorthand():
     with mock.patch("sys.argv", ["any.py", "--trainer.logger=False"]):
         cli = LightningCLI(TestModel, run=False)
     assert cli.trainer.logger is None
+
+
+def test_cli_auto_seeding():
+
+    with mock.patch("sys.argv", ["any.py"]):
+        from jsonargparse import ParserError
```

Comment:
Import this only once at the top of the test

Reply:
Debugging fragment, sry.